CodeEditor/Sources/CodeEditor/Utils/FeedbackSuggestion.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  FeedbackSuggestion.swift
+//  Themis
+//
+//  Created by Andreas Cselovszky on 25.01.23.
+//
+
+import Foundation
+
+public struct FeedbackSuggestion: Decodable, Equatable {
+    public let id = UUID()
+    public let exerciseId: Int
+    public let participationId: Int
+    public let code: String
```

Comment:
We can remove this attribute now because with my new PR the code will not be sent by default (Themis doesn't need it)